include/seqan3/range/container/dynamic_bitset.hpp

```diff
@@ -1082,13 +1088,13 @@ class dynamic_bitset
     //!\brief Direct access to the underlying bit field.
     constexpr bitfield * raw_data() noexcept
     {
-        return data;
+        return &data;
```

Comment:
how was this not caught before? ...

Reply:
Because it is a constexpr (implicitly inline) function and thus will not be "instantiated" The coverage says also 100% even though it isn't, because all those constexpr functions are hidden.

Reply:
How to trick the coverage :smile: